docs/Homebrew-and-Python.md

```diff
@@ -85,13 +87,21 @@ Since the system Python may not know which compiler flags to set when building b
 CFLAGS="-I$(brew --prefix)/include" LDFLAGS="-L$(brew --prefix)/lib" pip install <package>
 ```
 
-## Virtualenv
-
 **Warning!** When you `brew install` formulae that provide Python bindings, you should **not be in an active virtual environment.**
 
-Activate the virtualenv *after* you've brewed, or brew in a fresh terminal window. This will ensure Python modules are installed into Homebrew's `site-packages` and *not* into that of the virtual environment.
+Activate the virtualenv *after* you have installed your package with brew, or install brew's packages in a fresh terminal window. This will ensure Python modules are installed into Homebrew's `site-packages` and *not* into that of the virtual environment.
+
+## PEP688 (Python@3.12) and virtualenvs
+
+Starting with Python@3.12, Homebrew follows PEP688:
+https://peps.python.org/pep-0668/#marking-an-interpreter-as-using-an-external-package-manager
+
+To install Python packages system-wide, try `brew install xyz`, where xyz is the package you are trying to install.
```

Comment:
This seems (too) obvious?

Reply:
I have reorganised this part differently.